brainwave-prediction-app/GUI.py

```diff
@@ -132,19 +134,45 @@ def holding_pattern_window():
 # Define the layout for the Starting Page
 layout1 = [
     [sg.Button('Brainwave Reading', size=(20, 3)),
-     sg.Button('Transfer Data', size=(20, 3), disabled=True),
+     sg.Button('Transfer Data', size=(20, 3), disabled=False),

```

Comment:
Super minor again.  Maybe just remove the `disabled=False` to be consistent with the other buttons?  As it would be enabled by default.